### DISCUSSION TITLE
v2.13.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#6269] [Releases & Announcements]
## Changelog

- Ability to do video calls on Chatwoot through the Dyte Integration
- Native support for Microsoft as a provider in Email Inboxes
- Ability to use template variables while messaging
- Ability to send longer audio messages on Whatsapp Channel
- Support for incoming location messages in Telegram Channel
- Numerous bug fixes and enhancements

###
Thanks to @giquieu, @drexlma, @civilcoder55, @ooooooo-q, @matuszewskijan, @iprithvitharun, @jordan-brough  for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.13.0'>v2.13.0</a>.</em>

**Comment by sanjayws** [comment]
Congratulations on this new release. 

Is there any documentation on Dyte? This is super cool

Managed to get it working. Pretty damn sweet I must say.

**Reply by wladiarce** [reply]
Hi sanjayws,

Would you please share more details about how to make it work? I enabled it from the applications section, with the orgid and the api key, but from that point I don't see any further configuration neither more options.

Thanks!

**Reply by karthikindia** [reply]
its working fine, but no option to select any preset. it would be good to have this option else default meeting session is generated

**Reply by karthikindia** [reply]
also chatwoot  using version 1 need to update that to dyte version 2

**Comment by GuiVKDigital** [comment]
Can't see Microsoft config inside my chatwoot (updated right as I can see Dyte in integrations). Is there any activation that need to be done?

**Reply by sojan-official** [reply]
@GuiVKDigital  You will need to add environment variables related to Azure, We will have a doc put up soon 
cc: @tejaswinichile

**Reply by tejaswinichile** [reply]
https://www.chatwoot.com/docs/self-hosted/configuration/features/email-channel/azure-app-setup

**Comment by amaweco** [comment]
I loved the Video call feature 🔥
can we see Google meet and Zoom in future? since they're the most popular, so the customer can have the app launched automatically and avoid mic/camera configuration in browser to not work properly.

Again, you did a great job 👏

**Comment by yourbabyscan** [comment]
Any news when support for heroku-22 stack will be added? 

Build for heroku-22 stack fails at the moment with:
You are trying to install ruby-3.0.4 on heroku-22

Sticking on heroku-20 for the meantime.

**Reply by sojan-official** [reply]
@yourbabyscan  we have upgrade ruby version, which should support the heroku 22 stack. but we observed some issues with the slack integration when using the 22 stack.  if you aren't using the slack integration its safe to upgrade to 22 stack. we will be working on a fix meanwhile

**Comment by amaweco** [comment]
Can we have preset_name configured per inbox? as reference you can add that to "click to join" button event https://docs.dyte.io/api#/operations/add_participant

/integrations/dyte/add_participant_to_meeting

can be an option to select from the created presets, or just write the name explecitly

**Comment by sojan-official** [comment]
## 2.13.0 Dev Changelog

### API Endpoints

#### Additions
- only internal API additions

#### Deletions
- none

---

### Other Changes

- none
